src/ray/stats/metric_defs.h

```diff
@@ -159,3 +159,7 @@ static Gauge PendingPlacementGroups(
 static Histogram OutboundHeartbeatSizeKB("outbound_heartbeat_size_kb",
                                          "Outbound heartbeat payload size", "kb",
                                          {10, 50, 100, 1000, 10000, 100000});
+
+static Histogram GcsUpdateResourceUsageTime(
+    "gcs_update_resource_usage_time", "The average RTT of a UpdateResourceUsage RPC.",
+    "ms", {100, 200, 300, 400, 500, 600, 700, 800, 900, 1000}, {CustomKey});
```

Comment:
The histo buckets look a bit over-large, how about something more like {1, 2, 5, 10, 20, 40, 100, 200, 500, 1000} (exponential up-steps)?

Reply:
40 -> 50 for consistency, but yeah good call